src/test/groovy/com/jvm_bloggers/ObjectMother.groovy

```diff
@@ -0,0 +1,38 @@
+package com.jvm_bloggers
+
+import com.jvm_bloggers.entities.blog.Blog
+import com.jvm_bloggers.entities.blog_post.BlogPost
+
+import java.time.LocalDateTime
+
+import static com.jvm_bloggers.entities.blog.BlogType.PERSONAL
+
+class ObjectMother {
+
+    static Blog aBlog(Map params = [:]) {
+        String id = UUID.randomUUID().toString()
+        [
+                bookmarkableId    : "bookmarkableId $id".subSequence(0,29),
+                author            : 'author',
+                rss               : "rss $id",
+                url               : "url $id",
```

Comment:
At least let's make url and rss looking like valid :) e.g. `http://example.com/$id` etc.